chaudloader/src/hooks/stage1.rs

```diff
@@ -315,3 +365,29 @@ pub unsafe fn install_pck_load(game_env: &mods::GameEnv) -> Result<(), anyhow::E
     }
     Ok(())
 }
+
+/// Install optional BNK File load hook into the process.
+pub unsafe fn install_bnk_load(game_env: &mods::GameEnv) -> Result<(), anyhow::Error> {
+    unsafe {
+        if let Some(data) = game_env.sections.text {
+            // This pattern is enough to find the function in all releases of both collections (at 0x141CC27E0 Vol1 / 0x14302C310 Vol2 for latest releases)
```

Comment:
I would state the actual version numbers here as the "latest release" will change over time